file_paths.md

```diff
@@ -419,6 +420,43 @@ The package will install a service named `puppetmaster`, create a
         puppetmaster *                    # writeable by puppetmaster
             puppetmaster.pid
 
+# razor-server
+
+The package will install a service named `razor-server`, create a `razor`
+user and group, and run the service as the `razor` user.
+
+    /etc/puppetlabs/razor-server                                          /etc/razor
+        config.yaml
+        razor-toequebox.sh
+        shiro.ini
+    /opt/puppetlabs/bin
+        razor-admin
+        razor-binary-wrapper
+    /opt/puppetlabs/server/apps/razor-server
+        bin
+            jruby
+        sbin
+            torquebox
+        share
+            razor-server                                                  /opt/razor
+                bin
+                    razor-admin
+                    razor-binary-wrapper
+                brokers
+                lib
+                razor-server.env                                          /usr/share/razor-server/razor-server.env
+                shiro.ini
+                vendor
+            torquebox                                                    /opt/razor-torquebox
```

Comment:
I'm guessing that @smcclellan was pointing out that `/opt/razor-torquebox` needs to be preceded by one more space to align with the fields above.